docs/cpu_templates/cpuid-normalization.md

```diff
@@ -0,0 +1,77 @@
+# CPUID normalization
+
```

Comment:
Maybe worth mentioning that this is only for x86.

Reply:
added, thanks.